docs/features/exportpdf/README.md

```diff
@@ -0,0 +1,104 @@
+
+{@img assets/img/exportpdf_01.png Export to PDF button.}
+
+When enabled, it adds the **<img class="inline" src="%BASE_PATH%/assets/img/exportpdf-button.png" alt="Export to PDF toolbar button">Export to PDF** toolbar button. It sends data from editor to the endpoint maintained by [CKEditor Cloud Services](https://ckeditor.com/ckeditor-cloud-services/) and handles the response, saving it as a PDF file.
+
+## Feature overview
+
+Although CKEditor 4 is basically a web tool, sometimes you may want (or *need*) to use the content it generated somewhere else. While simple copy and paste may work in some cases, most probably it will not allow you to use some more advanced editor features like styled tables or lists. Thanks to the [Export to PDF](https://ckeditor.com/cke4/addon/exportpdf) plugin you can save the document you created to a downloadable file, preserving all the styles and structure in nearly 1:1 projection.
+
+## Supported features
+
+All the formatting and content structure you create using the editor is maintained in the output file. See the tips for {@link features/exportpdf/README#reaching-the-best-projection reaching the optimal projection} to learn how to configure the plugin and editor to best suite your needs.
+
+As for the compatibility, plugin works on all the browsers {@link guide/dev/browsers/README supported for CKEditor 4} except for Internet Explorers older than version 11.
+
+## Configuration
+
+### Reaching the best projection
+
+Due to the differences between browsers and operating systems it is not always possible to reach a perfect match between content in editor and PDF. However a lot can be done to ensure the differences will be hardly noticable.
+
+For example, make the editor width correspond with the chosen paper format for output file - e.g. for the `A4` format (which is used by the export service [by default](https://pdf-converter.cke-cs.com/docs#section/PDF-options/Page-format)) the editor width should be equal to `840px`. Then mind the margins; if they are changed for the PDF using [custom config](https://pdf-converter.cke-cs.com/docs#section/PDF-options/Margins), also update them for editor. To take things a step further you may experiment with {@link guide/dev/example_setups/README#document-editor Document Editor setup}.
+
+### Custom CSS rules
+
+Besides the inline styles, in order to mimic the editor content all the CKEditor 4 CSS rules are sent to the service. But let's say your company has a strict heading color policy: it has to be <span style="color:#4B0082">indigo</span> (`#4B0082`), definitely not <span style="color:#483D8B">dark slate blue</span> (`#483D8B`)! Fortunately you do not have to remember to change it every time. It is possible to attach your own custom CSS stylesheet via {@linkapi CKEDITOR.config.exportPdf_stylesheet CKEDITOR.config.exportPdf_stylesheet} configuration option or just add some styles with {@linkapi CKEDITOR.addCss} method.
```

Comment:
`{@linkapi CKEDITOR.config.exportPdf_stylesheet CKEDITOR.config.exportPdf_stylesheet}`

That will require some magic, since the plugin is not the part of presets we will need to copy it to `ckeditor4` directory for the time of docs generation so the API docs could be parsed :thinking: And having in mind that plugin code will be obfuscated we will need to extract API docs to a separate file for plugin dist probably? :thinking: :scream:

Reply:
Another option is putting all the config options in the `features` instead of in `API` section - AFAICS this is the way this problem was resolved for [Spreadsheet plugin](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_spreadsheets.html#automatically-convert-existing-tables)?

Reply:
Needless to say it's not a dream way of dealing with this problem.

Reply:
> Another option is putting all the config options in the `Features` instead of `API` section - AFAICS this is the way this problem was resolved for [Spreadsheet plugin](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_spreadsheets.html#automatically-convert-existing-tables)?

Yes, that was kind of a workaround there. When working on Export to PDF dist version building recently I already see that extracting comments is doable so we should be fine here :crossed\_fingers: